nodejs/builder/Dockerfile

```diff
@@ -8,6 +8,12 @@ FROM node:${NODE_BASE_IMG}
 ARG NODE_ENV
 ENV NODE_ENV $NODE_ENV
 
+ARG NPM_TOKEN
+ARG NPM_REGISTRY
+
+ENV NPM_TOKEN $NPM_TOKEN
```

Comment:
Since this could be a secret. It can be linked to builder podspec also at runtime. Rather than embedding in the image.

https://doc.crds.dev/github.com/fission/fission/fission.io/Environment/v1@v1.15.0

If variables are defined at the time of creation of builder, build.sh would use them during build.

Reply:
Although, fission currently doesn't have support to pass environment variables via CLI to builder.

Reply:
removed it